enhancements/sig-policy/72-content-to-ansilbe-job/README.md

```diff
@@ -0,0 +1,188 @@
+# Providing Context to the Ansible Automation for a Policy Violation 
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+ [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+The Open Cluster Management (OCM) Policy add-on Ansible integration should pass the additional information 
+into an Ansible job when triggering the creation. It means that the Ansible user can get more policy violation 
+information and use them in the Ansible playbook.
+
+## Motivation
+
+Policy Automation uses the `policyRef` field to match the target policy and monitor its non-compliance status. 
+A non-compliance policy can trigger its Policy Automation to create an Ansible job on the Ansible Tower, which 
+launches a predefined Ansible playbook against an inventory of hosts. During this creation, we can pass the 
+additional information to the Ansible job as `extra_vars`.
+
+The current limitation is that we only pass the non-compliant cluster as `extra_vars`, which isn't enough to 
+describe the policy violation status. The following fields could be added:
+1. The violated policy name
+2. The namespace of the root policy
+3. The hub cluster name
+4. The policy violation message
+5. The replicated policy status for each non-compliant cluster
+
+
+### Goals
+
+Adding the name of the violated policy, the root policy namespace, the hub cluster name, the violation message, and 
+the replicated policy status for each non-compliant cluster into the Ansible Job.
+
+### Non-Goals
+
+- None at this time
+
+## Proposal
+Policy automation runs on the hub cluster. It should not get the propagated policy status from the managed cluster,
+which requires Search or Observability dependency. The same design also applies to the hub of the cluster scenario.
```

Comment:
What does Observability have to do here ?

Reply:
I think it is mainly dependent on `Search` here, it can be updated if you think the current proposal isn't accurate.

Reply:
Yes. As I known,  view job link will be switched from policy page to search page, it won't related with Observability-grafana.